### PR TITLE
Fix the note format in upgrade-from-2.1-to-2.2.md

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### CODE REVIEW

File: versioned_docs/version-2.2.0/upgrade-from-2.1-to-2.2.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ Here is a list of Helm Chart changes:
 - value changed: `dashboard.serviceAccount` changed from `chaos-controller-manager` to `chaos-dashboard`
 - value changed: `webhook.FailurePolicy` changed from `Ignore` to `Fail`
 
-:::note For more information about the detailed description, see [README](https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md). :::
+:::note
+For more information about the detailed description, see [README](https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md). :::
```
Comment (Oreoxmt) [inline versioned_docs/version-2.2.0/upgrade-from-2.1-to-2.2.md:38]:
```suggestion
For more information about the detailed description, see [README](https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md).
:::
```

**Review by STRRL — COMMENTED** [review]
LGTM! ❤️

**Review by STRRL — APPROVED** [review]

**Pull request merged**